### PR TITLE
Configure only one project for CMakelists.txt node in ProjectOutline

### PR DESCRIPTION
**Pull request by vlavati**
### This changes visible behavior

The following changes are proposed:

- configure only project, that corresponds to CMakeLists.txt file node, instead of all projects

## The purpose of this change

The issue #3311 describes case with few projects in the ProjectOutline and it is possible to configure all projects, but there is no way to configure only a project that corresponds to the selected CMakeLists.txt. This PR changes the behavior of "configure" button to configure only one.

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -1876,7 +1876,13 @@ async function setup(context: vscode.ExtensionContext, progress?: ProgressHandle
         vscode.commands.registerCommand('cmake.outline.editCacheUI', () => runCommand('editCacheUI')),
         vscode.commands.registerCommand('cmake.outline.cleanRebuildAll', () => runCommand('cleanRebuildAll')),
         // Commands for outline items
-        vscode.commands.registerCommand('cmake.outline.configure', (what: ProjectNode) => runCommand('configure', what.folder, false, what.sourceDirectory)),
+        vscode.commands.registerCommand('cmake.outline.configure', async (what: object) => {
```
Comment (gcampbell-msft) [inline src/extension.ts:1879]: It'd be good if we could restrict this type more than just `object`, something that only allows for `ProjectNode` or `SourceFileNode`.

**Review by gcampbell-msft — CHANGES_REQUESTED** [review]
Everything looks really good except for my single comment. Thanks!

File: src/extension.ts (hunk 1 above)
Comment (vlavati) [reply src/extension.ts]: Thanks for review.
I've changed type annotation. Is it better?

Comment (gcampbell-msft) [reply src/extension.ts]: Yes! Everything looks good. Please add a Changelog entry and then I will approve and merge!

Comment (vlavati) [reply src/extension.ts]: Done

**Review by gcampbell-msft — APPROVED** [review]

**Pull request merged**